rr/display/display_manager.py

```diff
@@ -143,7 +143,7 @@ def create_display(self):
 
         for key in self._list:
             desc += " appsrc do-timestamp=true name=" + key + " format=time ! queue ! video/x-raw,width=" + str(w) + ",height=" + str(
-                h) + ",format=RGB,framerate=30/1,pixel-aspect-ratio=1/1" + " ! videoconvert ! videoscale ! video/x-raw,width=" + str(w) + ",height=" + str(h) + " ! mixer. "
+                h) + ",format=RGB,framerate=30/1,pixel-aspect-ratio=1/1" + " ! videoconvert ! videoscale ! video/x-raw,width=" + str(w) + ",height=" + str(h) + " ! perf ! mixer. "
```

Comment:
If you are adding `perf` to the pipelines, you need to guide the user in the README in how to install it

Reply:
we should remove perf